app/models/grouped_issue.rb

```diff
@@ -27,6 +27,16 @@ def users_affected
     subscribers.count
   end
 
+  def chart_data
+    data = []
+    issues = self.issues.where('created_at >= ? AND created_at <= ?', Date.today - 3.months, Date.today.end_of_day)
+    (Date.today - 1.months..Date.today.end_of_day).map(&:to_date).each do |date|
+      times = issues.select { |issue| issue.created_at.to_date == date }
+      data.push({value: times.size, date: date.strftime('%Y-%m-%d')})
+    end
+    data
```

Comment:
@ungureanu-bogdan-sorin let's remove this method by adding this gem: https://github.com/ankane/groupdate and utilise the method it provides to create sql queries that would give us the needed data. That means we can remove this once you add that one.